tests/test_minimal/test_tools/aws_tools.py

```diff
@@ -297,3 +297,150 @@ def test_submit_aws_batch_job_with_efs_mount():
     table.update_item(
         Key={"id": table_submission_id}, AttributeUpdates={"status": {"Action": "PUT", "Value": "Test passed."}}
     )
+
+
+class TestRcloneTransferBatchJob(TestCase):
+    """
+    To allow this test to work, the developer must create a folder on the outer level of their personal Google Drive
+    called 'testing_rclone_spikegl_and_phy' with the following structure:
+
+    testing_rclone_spikeglx_and_phy
+    ├── ci_tests
+    ├──── spikeglx
+    ├────── Noise4Sam_g0
+    ├──── phy
+    ├────── phy_example_0
```

Comment:
@pauladkisson Here is the new structure that mirrors that GIN specification test

Reply:
Ok, the files are up on my google drive [here](https://drive.google.com/drive/folders/1c9uYyhwQzOyQxYKvgr99jrlON8rDR2dO?usp=drive_link). Should be matching the structure you outlined, but if there are any problems just lmk.